src/fees.ts

```diff
@@ -1,35 +1,33 @@
-/** 
+/**
  * This module provides functions for calculating & validating
  * transaction fees.
- * 
+ *
  * @module fees
  */
 
-import BigNumber from 'bignumber.js';
+import BigNumber from "bignumber.js";
 
-import {
-  P2SH,
-  estimateMultisigP2SHTransactionVSize,
-} from "./p2sh";
+import { P2SH, estimateMultisigP2SHTransactionVSize } from "./p2sh";
 import {
   P2SH_P2WSH,
   estimateMultisigP2SH_P2WSHTransactionVSize,
 } from "./p2sh_p2wsh";
-import {
-  P2WSH,
-  estimateMultisigP2WSHTransactionVSize,
-} from "./p2wsh";
-import {ZERO} from "./utils";
+import { P2WSH, estimateMultisigP2WSHTransactionVSize } from "./p2wsh";
+import { ZERO } from "./utils";
+
+// Without this, BigNumber will report strings as exponentials. 16 places covers
+// all possible values in satoshis.
+BigNumber.config({ EXPONENTIAL_AT: 16 });
```

Comment:
Is there any concern about this impacting uses of BigNumber that aren't for satoshis?